hackathon_site/dashboard/frontend/src/slices/event/teamDetailSlice.ts

```diff
@@ -30,6 +30,46 @@ interface RejectValue {
     message: any;
 }
 
+interface idProvidedParameters {
+    profileId: number;
+    idProvided: boolean;
+    attended: boolean;
+}
+
+export const updateParticipantIdProvided = createAsyncThunk<
+    Profile,
+    idProvidedParameters,
+    { state: RootState; rejectValue: RejectValue; dispatch: AppDispatch }
+>(
+    `${teamDetailReducerName}/updateParticipantIdProvided`,
+    async ({ profileId, idProvided, attended }, { rejectWithValue, dispatch }) => {
+        try {
+            const response = await patch<Profile>(`/api/event/profiles/${profileId}/`, {
+                id_provided: !idProvided,
```

Comment:
I feel like we should handle this logic in the TeamInfoTable component, keep this thunk generalized in case we want to use it elsewhere. 
So just pass in `idProvided` and handle the toggle logic in the checkbox onclick function.

Reply:
fixed in teamDetailSlice...

Reply:
fixed